### ISSUE TITLE
boards.txt not found

### ISSUE THREAD

**Issue by laica10** [Sloeber/arduino-eclipse-plugin#833]
Just installed sloeber complete pack on macOS I suppose it's version 4 but when I start sloeber it just gives me an error wich says: "No boards.txt files found in the arduino hardware folders". 
Someone encountered this problem or solved it yet? (I7 2.6GHz, 16GB RAM, 64x)

**Comment by jantje** [comment]
Can you provide a image of the error.
Have you looked at the video's?
https://www.youtube.com/watch?v=zc2JLbtbI7Q&list=PLCSjf7QmP3TDPSjT_5UIp0DHXzKdhIQsF

As an independent open source project we value your submission, but we ask for support, either by helping us out coding (yeah, we do understand it does require time) or a Patreon contribution (starting from as low as 1$ a month): this allows us to support people who support us back!
As a supporter you can drag our attention and prioritize your submission...
please become a supporter!
https://www.patreon.com/jantje

**Comment by seamaster101** [comment]
I have similar problem with teensy boards. They are not installed by default, and even when I follow old instructions, it shows up, in the board selection, the is not upload protocol selection available and will not let me create teensy project
![screen shot 2017-09-28 at 10 37 10 pm](https://user-images.githubusercontent.com/7897023/31002137-a22d5550-a49d-11e7-908f-fc131f5cca6e.png)

**Comment by jantje** [comment]
@seamaster101 I think you need to enlarge the window as there are more options which have not yet been set.
As to the uploader you will have to copy the programmers.txt as explained in #826.
As to the teensy install: what do you mean with "and even when I follow old instructions'

**Comment by uzi18** [comment]
@jantie: is it possible to add there always shown slider as it sometimes
hide some options below in this window.

29.09.2017 2:38 PM "jantje" <notifications@github.com> napisał(a):

**Comment by jantje** [comment]
If you know how, feel free.

**Comment by uzi18** [comment]
Ok will investigate it.

29.09.2017 3:21 PM "jantje" <notifications@github.com> napisał(a):

If you know how, feel free.

—
You are receiving this because you commented.

Reply to this email directly, view it on GitHub
<https://github.com/Sloeber/arduino-eclipse-plugin/issues/833#issuecomment-333124411>,
or mute the thread

**Comment by jantje** [comment]
It should be somewhere in here https://github.com/Sloeber/arduino-eclipse-plugin/blob/master/io.sloeber.ui/src/io/sloeber/ui/project/properties/BoardSelectionPage.java

**Comment by laica10** [comment]
![image](https://user-images.githubusercontent.com/1689634/31024951-02f30242-a541-11e7-8eae-a88549239e22.png)
This is the error I'm having, sorry for the delayed response but I was really busy.

**Comment by mjmeijer** [comment]
Reinstall and move Sloeber to Applications folder first before launching fir first time. Before launching enable Install applications from anywhere in Security preference

**Comment by laica10** [comment]
@mjmeijer, Thanks! It worked with just moving Sloeber to applications. Note that u can't enable install applications from anywhere anymore in security preferences.

**Comment by seamaster101** [comment]
@jantje  Reporting back that my issue was resolved too. There is more hidden window, but not an indication "like a scroll-bar" that some part can not be seen.  
I also copied programmers.txt as indicated above and that resolved "upload protocol: default" only issue.
I only need to find out how to make my existing Teensy project compile. Right now there is no Arduino option in the properties of imported project, but issue for another day and it is not related to the above,
thank you so much!
